mail_tracking/static/src/xml/mail_tracking.xml

```diff
@@ -43,6 +43,9 @@
 <t t-extend="mail.thread.message">
     <t t-jquery="span[t-attf-class='oe_partner_follower']" t-operation="append">
         <t t-set='tracking' t-value='widget.partner_trackings[partner[0]]'/>
+        <t t-if="!tracking">
+            <t t-set='tracking' t-value="['unknown']"/>
```

Comment:
You can put it in line 45: `widget.partner_trackings[partner[0]] || ['unknown']`